test_util/cluster_api.py

```diff
@@ -200,6 +200,7 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
                 configured if its value is "true".
             provider: onprem, azure, or aws
             auth_enabled: True or False
+            mesos_user: account that mesos is running under
```

Comment:
Hmmm, my naming decision was influenced by https://github.com/dcos/dcos/pull/772 which adds a default_user option. So, how about we call this os_default_framework_user?